docs/geedocs/5.3.4/sources/answer/releaseNotes/relNotesGEE5_3_4.rst.txt

```diff
@@ -28,8 +28,9 @@ new PostgreSQL version used by Open GEE 5.3.4.
 
 -[ Resolved Issues ]-
 
-      Number  Description  Resolution  Issue number  Issue Description
-Issue Resolution
+      Number  Description  Resolution  1604  WMS capability doesn't
```

Comment:
Is this change intended to be on this PR? It doesn't look relevant.

Also, are there release notes changes for the minify feature?

Reply:
It's not, but I'm not sure what to do about it.  Looks like a pull request merged a few days ago updated the release notes RST but forgot to regenerate the HTML and .rst.txt 

https://github.com/google/earthenterprise/commit/1985695bc580c15546640232cd993fdc43565440